pallets/multisig/src/lib.rs

```diff
@@ -945,24 +889,27 @@ impl<T: Trait> Module<T> {
                     );
                 }
                 if proposal_details.approvals >= Self::ms_signs_required(&multisig) {
```

Comment:
Indentation could be reduced if we return here:
```
if proposal_details.approvals < Self::ms_signs_required(&multisig) {
    return Ok(());
}
...
```

Reply:
@vkomenda This code is inside a `match`. Returning from here will return from the full function, not just the match. Unfortunately, rust doesn't allow breaking from a match without ugly hacks.

Reply:
In a situation like this I'd instead opt to move the stuff in the `if`'s then block into a function

Reply:
Yeah, we can do it in a future refactoring maybe. We can live with some indentation for now :)